polynote-runtime/src/main/scala/polynote/runtime/ReprsOf.scala

```diff
@@ -9,7 +9,9 @@ import shapeless.HNil
 import scala.collection.GenSeq
 import scala.concurrent.Future
 
-trait ReprsOf[T] extends (T => Array[ValueRepr]) with Serializable
+trait ReprsOf[T] extends Serializable {
+  def apply(value: T): Array[ValueRepr]
+}
```

Comment:
I had to remove the `extends (T => Array[ValueRepr])` because it turns out that Scala will consider instances for `T` to be an implicit conversion candidate for `T => Array[ValueRepr]` and this interferes with `applyDynamic*` methods for `PythonObject`.